typescript/src/entities/email_address.ts

```diff
@@ -7,7 +7,7 @@ export class EmailAddress extends Entity {
   protected static singularName: string = "emailAddress";
   protected static pluralName: string = "emailAddresses";
 
-  @EmailAddress.property({type: EmailAddress})
+  @EmailAddress.property({type: Date})
```

Comment:
how come?

Reply:
previously was Wrong, Date was the right one which maps to archived